### PR TITLE
Create CGINS-ADCPAM-55979__20200709.csv

### PR DESCRIPTION
**Pull request by cdobs**
This creates a "calibration csv" for a newly acquired glider DVL transducer. The transducer in DVL bay 541 was recently replaced and since we now track DVL transducer SNs instead of DVL bay numbers, we need to add a calibration csv for this new transducer. Since the transducers do not actually get calibrated, we use acquisition or vendor service dates for the date to be used in the cal csv.

The transducer swap can be tracked in: $/Project_Files/Records/Platform_Records/Gliders/Coastal_glider/DVL/CSR_5878_DVL-541_2020-07-09.pdf

And glider 387's FCP: $/Project_Files/Records/Platform_Records/3304-00150-00129-A.zip/387_Pioneer_15a_FCP.cap CTRL-F for the text "PS0" to show the output of when the DVL is queried for its transducer SN

### CODE REVIEW

File: calibration/ADCPAM/CGINS-ADCPAM-55979__20200709.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+655979,CC_scale_factor1,0.61,
+655979,CC_scale_factor2,0.61,
+655979,CC_scale_factor3,0.61,
+655979,CC_scale_factor4,0.61,
```
Comment (rtravis-whoi) [inline calibration/ADCPAM/CGINS-ADCPAM-55979__20200709.csv:5]: I think the cal date on the csv filename (and the vendor doc) should be 2020-06-24, which is the date on the Configuration Summary & Cert of Conformance in the vendor docs.

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
I think the DVL cal date on the filename should be 2020-06-24, which is the date on the Configuration Summary & Cert of Conformance in the vendor docs.

File: calibration/ADCPAM/CGINS-ADCPAM-55979__20200624.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+655979,CC_scale_factor1,0.61,
+655979,CC_scale_factor2,0.61,
+655979,CC_scale_factor3,0.61,
+655979,CC_scale_factor4,0.61,
```
Comment (rtravis-whoi) [inline calibration/ADCPAM/CGINS-ADCPAM-55979__20200624.csv:5]: looks good!

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/ADCPAM/CGINS-ADCPAM-55979__20200624.csv (hunk 2 above)
Comment (snwhite) [reply calibration/ADCPAM/CGINS-ADCPAM-55979__20200624.csv:5]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**